src/aggregator/aggregator/elem_base.go

```diff
@@ -258,6 +262,11 @@ type flushMetrics struct {
 	forwardLags map[forwardKey]tally.Histogram
 }
 
+type writeMetrics struct {
+	writes        tally.Counter
+	updatedValues tally.Counter
```

Comment:
i migrated the existing `updatedValues` metric to include a `list-type` tag as well to be consistent. it's backwards compatible